client/router.js

```diff
@@ -198,6 +198,12 @@ Router.prototype.ready = function() {
 };
 
 Router.prototype.subsReady = function() {
+  var callback, args = Array.prototype.slice.call(arguments);
```

Comment:
This seems like `Array.slice` returning two items :) Better to write like this.

~~~js
var callback = null;
var args = Array.prototype.slice.call(arguments);
~~~

And how about using `_.toArray()` instead `Array.slice`